lib/src/js/polymer_base_dart_es6.html

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+for details. All rights reserved. Use of this source code is governed by a
+BSD-style license that can be found in the LICENSE file.
+-->
+<script>
+  Polymer.Dart = Polymer.Dart || {};
+  Polymer.Dart.ES6 = Polymer.Dart.ES6 || {};
+
+
+  (function(ES6) {
+
+    ES6.Unsupported = {};
+
+    ES6.dartGetter = function() {
+      // PLACEHOLDER : THIS IS DEFINED IN THE DART SIDE
```

Comment:
Maybe throw an error inside here, mentioning that this function should have been overriden?